ui/src/components/admin/liveSermon/AdminLiveSermonContainer.js

```diff
@@ -194,6 +203,18 @@ export default function AdminLiveSermonContainer(props) {
     getData();
   }, [getData]);
 
+  useEffect(() => {
+    if (streamStartTime && streamEndTime) {
+      const startTime = DateTime.fromISO(streamStartTime);
+      const endTime = DateTime.fromISO(streamEndTime);
+      if (startTime >= endTime) {
+        setStreamPeriodInvalid(true);
+        return;
+      }
+    }
+    setStreamPeriodInvalid(false);
```

Comment:
also no need to `setStreamPeriodInvalid` to `false` since you've initialised it as `false` anyway

Reply:
For this change and the change 'no need to return here' have to be put because it will always be set to true and I cannot update the sermon